src/cmps/button/Button.css

```diff
@@ -0,0 +1,58 @@
+
+.button{
+    min-width: 130px;
```

Comment:
mmm, I didn't mention the rem unit.
start with reading and understanding: https://snook.ca/archives/html_and_css/font-size-with-rem

later on we will add a ticket for refactoring and removing all `px` units

Reply:
i know how to use in rem unit...